### PR TITLE
Yutong Li's pull request

### PR DESCRIPTION
**Pull request by victor-YT**
This is my pull request. I do GFnet.

### CODE REVIEW

**Comment by victor-YT** [comment]
It's a GFNet model implement. Have modules, utils, predict, train, dataset and README.

**Comment by victor-YT** [comment]
Hi there, I just fix some bugs

**Comment by wangzhaomxy** [comment]
**<This is an initial inspection, no action is required at this point.>**

**File Organizing:** Well-organized files.

**Problem Solving:**

- The algorithm solves the problem appropriately. 
- Accuracy in testing dataset: 0.7591. 

**Model and functions:**

- It correctly uses PyTorch to construct the GFNet models and functions.
- With data augmentation.

**Code design:** Good.

**Code comment and docstring:**

- No code comments 
- Minimal function docstrings

**Difficulty:** Hard.

**Additional Comments:**

- Meanless commits like "update xxxx", "fix bug", etc.
- No conclusion in the report.

**Comment by aniketgupta17** [comment]
Observational Feedback

Pull Request:
Correctly created the Pull request from Topic Recognition Branch .
The pull request has no description about the file structure .
No previous feedback incorporated . 

File Organizing: Well-organized files.

Commit Log:
Commit messages are irrelevant. 
Commits should  demonstrate a  more logical development flow.
Commits are regularly made.

Documentation:
The README file  should be more informative and meaningful . 
Code comments and docstrings  are  not included.
Proper GitHub markdown formatting is used.
Can add about  future improvements in the ReadMe.

**Comment by victor-YT** [comment]
Hi mate, thanks for the feedback but I have two small questions.
From the previous pull request I have described the file structure. Do we need to go into a very detailed description of each file?
And I added some code comments and docstrings in these files and also the conclusion in README according the previous feedback. Is this not acceptable? Because the new feedback says "No previous feedback incorporated ." and "Code comments and docstrings are not included."
> It's a GFNet model implement. Have modules, utils, predict, train, dataset and README.

**Comment by aniketgupta17** [comment]
I have updated the comment , I would like you to look at different Pull Requests where people give a detailed description of the file Structure  and results achieved currently it just says "This is my pull request. I do GFnet."

**Comment by victor-YT** [comment]
I see. Thanks for the feedback. I just did some change about the pull request comment.

**Comment by gayanku** [comment]
## Marking
<table><tbody>
  <tr><td colspan='3'><b>Good/OK/Fair Practice (Design/Commenting, TF/Torch Usage)</b></td> </tr>
	<tr><td></td><td>Good design and implementation. </td><td></td></tr>
	<tr><td></td><td>Spacing and comments. </td><td></td></tr>
	<tr><td></td><td>Header blocks. </td><td></td></tr>
	
  <tr><td colspan='3'><b>Recognition Problem</b></td></tr>
	<tr><td></td><td>Good solution to problem. </td><td></td></tr>
	<tr><td></td><td>Driver Script present. </td><td></td></tr>
	<tr><td></td><td>File structure present. </td><td></td></tr>
	<tr><td></td><td>Good Usage & Demo & Visualisation & Data usage. </td><td></td></tr>
	<tr><td></td><td>Module present. </td><td></td></tr>
	<tr><td></td><td>Commenting missing. Minimal doc strings.</td><td>-1</td></tr>
	<tr><td></td><td>No Data leakage found. Uses Data Aug</td><td></td></tr>
	<tr><td></td><td>Difficulty : Hard.  GFNet (Hard Difficulty)</td><td></td></tr>
	
  <tr><td colspan='3'><b>Commit Log</b></td></tr>
	<tr><td></td><td>No Meaningful commit messages. </td><td>-2</td></tr>
	<tr><td></td><td>Good Progressive commits. </td><td></td></tr>
	
  <tr><td colspan='3'><b>Documentation</b></td></tr>
	<tr><td></td><td>Readme :Good. </td><td></td></tr>
	<tr><td></td><td>Model/technical explanation :Good. </td><td></td></tr>
	<tr><td></td><td>Description and Comments :Good. </td><td></td></tr>
	<tr><td></td><td>Markdown used and PDF submitted. PDF Checked.</td><td></td></tr>
	
  <tr><td colspan='3'><b>Pull Request</b></td></tr>
	<tr><td></td><td>Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch). </td><td></td></tr>
	<tr><td></td><td>No Feedback required. </td><td></td></tr>
	<tr><td></td><td>Request Description is adequate. </td><td>-1</td></tr>
  <tr><td colspan='2'><b>TOTAL</b></td><td>-4</td></tr>
</tbody></table>

Marked as per the due date and changes after which aren't necessarily allowed to contribute to grade for fairness.
_Subject to approval from Shakes_
